client/lib/api/who_service.dart

```diff
@@ -24,7 +25,7 @@ class WhoService {
     final postBody = jsonEncode(req.toProto3Json());
 
     final url = '$serviceUrl/putDeviceToken';
-    final response = await http.post(url, headers: headers, body: postBody);
+    final response = await httpPost(url, headers, postBody);
```

Comment:
General thought is that the `http.post` APIs will have a lot of thought that's gone in to them. So my suggestion was to use the same API for `httpPost`. It also benefits users as they can use the same syntax and it benefits reviewers as there's fewer changes switching from one to the other. So you'd expect the new line to be:

```suggestion
    final response = await await httpProxy.post(url, headers: headers, body: postBody);
```